mantidproject/mantid
# Move plotting functions for ipython notebooks to an import

Closes #14509 and #14737 

**Requires testing in Linux**

We had feedback through Tim Charlton that some users found the plotting functions in ipython notebooks intimidating. To solve this I have moved the plotting functions into a package called `MantidIPython` and import this in the notebooks. The package may be expanded in the future to contain other useful functions for notebooks, so the name does not contain "plot".

**To Test:**
- Generate an ipython notebook from the Reflectometry (Polref) Interface by following the example workflow in the [interface's documentation](http://docs.mantidproject.org/nightly/interfaces/ISIS_Reflectometry.html?highlight=reflectometry%20interface) and ticking the "Output Notebook" checkbox before "Process".
- The "save notebook" dialog may appear under the MantidPlot window, and a file extension is not automatically appended to a filename. These problems are being solved in #14775.
- Run ipython notebook using `/opt/mantidunstable/bin/mantidpython notebook`. Previously an error would occur about ipython module not being found.
- Open the notebook and ensure that it contains `from MantidIPython import *` and that plots are output when you click "Cell -> Run All" from the top of the window.
